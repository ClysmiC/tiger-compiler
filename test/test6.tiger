let
        var a, b : int;
in
        a := 1;
        while(a < 100) do
               a + := b;
        enddo;
        printi(a);
end
