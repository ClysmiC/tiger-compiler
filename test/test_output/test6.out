LET
VAR
ID
COMMA
ID
COLON
INT
SEMI
IN
ID
ASSIGN
INTLIT
SEMI
WHILE
LPAREN
ID
LESSER
INTLIT
RPAREN
DO
ID
PLUS

(Parser error) Line: 6
a +<---
Unexpected token found: PLUS
Expected token: ASSIGN

ASSIGN
ID
SEMI
ENDDO
SEMI
ID
LPAREN
ID
RPAREN
SEMI
END
EOF

Unsuccessful parse